db/migrations/0003_alter_order_created_at.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-11-18 11:50
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0002_user_order_ticket_alter_movie_actors_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='order',
+            name='created_at',
+            field=models.DateTimeField(),

```

Comment:
The `created_at` field in the `Order` model should have `auto_now_add=True` to automatically set the field during creation, as specified in the task description. This migration removes that setting, which is not recommended. Please ensure `auto_now_add=True` is included.